ios/Podfile.lock

```diff
@@ -228,6 +228,8 @@ PODS:
     - React-Core
   - react-native-netinfo (4.7.0):
     - React
+  - react-native-orientation-locker (1.5.0):
```

Comment:
how this related to this PR?

Reply:
it's not, will remove. thanks for spotting 👍